src/wallet/load.cpp

```diff
@@ -42,16 +42,16 @@ bool VerifyWallets(interfaces::Chain& chain, const std::vector<std::string>& wal
     std::set<fs::path> wallet_paths;
 
     for (const auto& wallet_file : wallet_files) {
-        WalletLocation location(wallet_file);
+        fs::path path = fs::absolute(wallet_file, GetWalletDir());
```

Comment:
aac063f28e4ed48180c2ba7a51706af969ca1e55, nit
```suggestion
        const fs::path path = fs::absolute(wallet_file, GetWalletDir());
```